src/plugin.ts

```diff
@@ -60,6 +60,9 @@ export class WebpackNoModulePlugin {
                 const nomodule = this._config.filePatterns.some(pattern => minimatch(s.attributes.src, pattern));
                 if (nomodule) {
                     s.attributes.nomodule = true;
+                    if (s.attributes.type === 'module') {
+                        delete s.attributes.type;
+                    }

```

Comment:
should we support this behavior, or the opposite? I'm not sure of a scenario in which `module` and `nomodule` should coexist, but do you think its better to delete the `module` rather than skip adding `nomodule`?

Reply:
Setting html-webpack-plugin option `scriptLoading='module'` adds `type='module'` to all entries. 

I believe the correct thing here is to remove `type` attribute. Setting both `type='module'`and `nomodule` does not make sense (`nomodule` will be ignored and the script will be loaded in modern browsers that support modules).

Am i missing something here? :-)

Reply:
Ah, I see. yeah, if the `module` tag is simply blanket applied, I think this is the correct solution. I was under the impression it was attached via pattern like the nomodule tag.